code/game/gamemodes/clock_cult/clock_structure.dm

```diff
@@ -172,29 +172,39 @@
 				amount -= MIN_CLOCKCULT_POWER
 	var/apcpower = accessable_apc_power()
 	while(apcpower >= MIN_CLOCKCULT_POWER && amount >= MIN_CLOCKCULT_POWER)
-		if(target_apc.cell.use(MIN_CLOCKCULT_POWER))
+		if(target_apc.cell && target_apc.cell.use(MIN_CLOCKCULT_POWER))
```

Comment:
If it has accessable_apc_power() target_apc is valid. Actually, I don't even need this check, because accessable_apc_power() checks for a cell to get the amount of power the apc has.